nextmv/nextmv/cloud/application.py

```diff
@@ -1280,6 +1372,53 @@ def new_batch_experiment_with_result(
 
         return self.batch_experiment_with_polling(batch_id=batch_id, polling_options=polling_options)
 
+    def new_ensemble_defintion(
+        self,
+        id: str,
+        run_groups: list[RunGroup],
+        rules: list[EvaluationRule],
+        name: Optional[str] = None,
+        description: Optional[str] = None,
+    ) -> EnsembleDefinition:
+        """
+        Create a new ensemble definition.
+
+        Parameters
+        ----------
+        id: str
+            ID of the ensemble defintion.
+        name: Optional[str]
+            Name of the ensemble definition.
+        description: Optional[str]
+            Description of the ensemble definition.
+        run_groups: Optional[list[RunGroup]]
+            Information to facilitate the execution of child runs.
+        rules: Optional[list[EvaluationRule]]
+            Information to facilitate the selection of
+            a result for the ensemble run from child runs.
```

Comment:
Parameters should be in the same order they are expected in the function arguments.